features/raadpleeg-bewoning-met-periode/bewoning-vaste-volgorde.feature

```diff
@@ -32,6 +32,29 @@ Functionaliteit: De sortering van bewoners op vaste volgorde
         |           000000024 |
         |           000000012 |
 
+    @valideer-volgorde
+    Scenario: De mogelijke bewoners worden gesorteerd op datum aanvang adreshouding
```

Comment:
ik denk dat het volgende een betere scenario titel is: De mogelijke bewoners in de gevraagde periode hebben verschillende datum aanvang adreshouding